src/asmjit/core/operand.h

```diff
@@ -1135,12 +1135,10 @@ public:                                                                      \
 //! signature.
 #define ASMJIT_DEFINE_FINAL_REG(REG, BASE, TRAITS)                           \
 public:                                                                      \
-  enum : uint32_t {                                                          \
-    kThisType  = TRAITS::kType,                                              \
-    kThisGroup = TRAITS::kGroup,                                             \
-    kThisSize  = TRAITS::kSize,                                              \
-    kSignature = TRAITS::kSignature                                          \
-  };                                                                         \
+  static constexpr auto kThisType = TRAITS::kType;                           \
```

Comment:
Here should not be `auto`, just use the right type here

Reply:
Changed.